xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamNavigator.cpp

```diff
@@ -1026,60 +1026,56 @@ void CDVDInputStreamNavigator::SetAudioStreamName(AudioStreamInfo &info, const a
   switch(audio_attributes.audio_format)
   {
   case DVD_AUDIO_FORMAT_AC3:
-    info.name += " AC3";
+    info.codecDesc = "AC3";
     info.codecName = "ac3";
     break;
   case DVD_AUDIO_FORMAT_UNKNOWN_1:
-    info.name += " UNKNOWN #1";
+    info.codecDesc = "UNKNOWN #1";
     break;
   case DVD_AUDIO_FORMAT_MPEG:
-    info.name += " MPEG AUDIO";
+    info.codecDesc = "MPEG AUDIO";
     info.codecName = "mp1";
     break;
   case DVD_AUDIO_FORMAT_MPEG2_EXT:
-    info.name += " MP2 Ext.";
+    info.codecDesc = "MP2 Ext.";
     info.codecName = "mp2";
     break;
   case DVD_AUDIO_FORMAT_LPCM:
-    info.name += " LPCM";
+    info.codecDesc = "LPCM";
     info.codecName = "pcm";
     break;
   case DVD_AUDIO_FORMAT_UNKNOWN_5:
-    info.name += " UNKNOWN #5";
+    info.codecDesc = "UNKNOWN #5";
     break;
   case DVD_AUDIO_FORMAT_DTS:
-    info.name += " DTS";
+    info.codecDesc = "DTS";
     info.codecName = "dts";
     break;
   case DVD_AUDIO_FORMAT_SDDS:
-    info.name += " SDDS";
+    info.codecDesc = "SDDS";
     break;
   default:
-    info.name += " Other";
+    info.codecDesc = "Other";
     break;
   }
 
   switch(audio_attributes.channels + 1)
   {
   case 1:
-    info.name += " Mono";
+    info.codecDesc += " Mono";
     break;
   case 2:
-    info.name += " Stereo";
+    info.codecDesc += " Stereo";
     break;
   case 6:
-    info.name += " 5.1";
+    info.codecDesc += " 5.1";
     break;
   case 7:
-    info.name += " 6.1";
+    info.codecDesc += " 6.1";
     break;
   default:
-    char temp[32];
-    snprintf(temp, sizeof(temp), " %d-chs", audio_attributes.channels + 1);
-    info.name += temp;
+    info.codecDesc += StringUtils::Format(" {:d}-chs", audio_attributes.channels + 1);
```

Comment:
use localized string
```suggestion
    info.codecDesc += StringUtils::Format(" {:d} {}", audio_attributes.channels + 1,
                                          g_localizeStrings.Get(10127));
```
and add the file path to the relative id on
addons\resource.language.en_gb\resources\strings.po

Reply:
there is no space problem codecDesc is shown on the second line
![immagine](https://github.com/user-attachments/assets/a1949326-682b-4eb7-a831-c63e7661d353)

Reply:
if the problem the popup only
then should be found a way to improve the popup only without affect other parts
for example why the popup should have this big icon ? could be removed and save a lot of space for text
where the first line can show "Audio: english"
and second line the description